lib/Doctrine/ORM/Proxy/Factory/ProxyFactory.php

```diff
@@ -47,7 +47,7 @@
     const AUTOGENERATE_EVAL = 3;
 
     /**
-     * @param array<Doctrine\ORM\Mapping\ClassMetadata> $classMetadataList
```

Comment:
https://i.redditmedia.com/0UfebEHWhsAa6CyTcxXnHzSnVdaw3x2TRFJjdT_-CGw.jpg?w=538&s=724ae8ed0e8391ab9f3f36c4458cb43c

Reply:
First requires to be fixed in known code analysis tools, then we can use it 😁